packages/core/src/card/card.element.ts

```diff
@@ -0,0 +1,64 @@
+import { property, globalStyle } from '@cds/core/internal';
+import { CdsInternalPanel } from '@cds/core/internal-components/panel';
+import { css } from 'lit-element';
+import { styles } from './card.element.css.js';
+
+export const CdsCardTagName = 'cds-card';
+
+/**
+ * Web component card.
+ *
+ * ```typescript
+ * import '@cds/core/card/register.js';
+ * ```
+ *
+ * ```html
+ * <cds-card>
+     <div cds-layout="vertical gap:md">
+       <div>
+         <p cds-text="h4">Card Title</p>
+       </div>
+
+       <cds-divider cds-card-stretch-full></cds-divider>
+
+       <div class="body" cds-text="p1" style="overflow:auto">
+         ${longerContent}
+       </div>
+
+       <cds-divider cds-card-stretch-full></cds-divider>
+
+       <div class="footer">
+         <cds-button action="flat"> View </cds-button>
+       </div>
+     </div>
+ * </cds-card>
+ * ```
+ *
+ * @element cds-card
+ * @slot - For projecting card content
+ * @cssprop --width
+ * @cssprop --height
+ * @cssprop --color
+ * @cssprop --overflow
+ * @cssprop --overflow-x
+ * @cssprop --overflow-y
+ * @cssprop --background
+ * @cssprop --border
+ * @cssprop --border-radius
+ * @cssprop --padding
+ * @cssprop --box-shadow
```

Comment:
Probably need to note the `--card-remove-margin` here. We also need to verify all the other properties listed. It seems like we have too many overflow properties.